src/app/shared/components/card/card.component.js

```diff
@@ -13,8 +13,7 @@ class Card extends Component {
 
     render() {
         return (
-            <div className={`card-component
-                            ${this.props.hoverEnabled ? 'hover-enabled' : ''}`}
+            <div className={`card-component${this.props.hoverEnabled ? ' hover-enabled' : ''}`}
```

Comment:
Please change to this:
<div className={`card-component ${this.props.hoverEnabled ? 'hover-enabled' : ''}`}